policyengine_us/parameters/calibration/gov/cbo/income_by_source.yaml

```diff
@@ -94,6 +248,75 @@ qualified_dividend_income:
   2033-01-01: 536_400_000_000
   2034-01-01: 562_000_000_000
   2035-01-01: 586_900_000_000
+  # Extended projections (2036-2100) using SSA GDP + equity premium
+  # Methodology: Applied SSA GDP growth rates + 0.5% equity premium
```

Comment:
source for the 0.5%?

if some income sources grow faster than gdp, others have to grow more slowly

Reply:
This is no longer in there